src/Bridge/Symfony/Bundle/Resources/config/metadata/metadata.xml

```diff
@@ -69,6 +69,10 @@
             <argument type="service" id="api_platform.metadata.property.metadata_factory.serializer.inner" />
         </service>
 
+        <service id="api_platform.metadata.property.metadata_factory.constructor" class="ApiPlatform\Core\Metadata\Property\Factory\ConstructorPropertyMetadataFactory" decorates="api_platform.metadata.property.metadata_factory" decoration-priority="20" public="false">
```

Comment:
Could we use 30 as a priority as well?

Reply:
Actually 35 makes even more sense (as the serializer `SerializerPropertyMetadataFactory` will potentially modify this accessibility) !